### ISSUE TITLE
Changed Viewport after changed dpi in print

### ISSUE THREAD

**Issue by johnnyblasta** [origo-map/origo#1547]
**Describe the bug**
When the DPI is changed in print control the viewport on the map is changed when returning from print.

**To Reproduce**
![ChangedViewport](https://user-images.githubusercontent.com/48793206/171420759-3ebf4e34-dea1-4d1c-8eec-08111f1bc0e4.gif)

**Expected behavior**
Would expect that the map shows the same area as before when returning from print when all that was done was changing the DPI.

**Comment by steff-o** [comment]
I think this is actually how it always have worked (but that does not make it right). Question is what should happen when leaving print preview:
1. Return to the extent the map had before entering preview. This could be confusing for the user if the user expects the print preview and the map to behave as one. 
2. Keep the center point and zoom level from print preview. As the form factor differs between paper and screen the exact extent can not be displayed.  This could be confusing if user has recomposed the extent in print preview and the user expects that what happens in the print preview stays in the print preview.

Basically it depends on what the user expects and I think that the displayed behavior is the result of lack of decision what the user should expect.

**Comment by johnnyblasta** [comment]
> I think this is actually how it always have worked (but that does not make it right). Question is what should happen when leaving print preview:
> 
> 1. Return to the extent the map had before entering preview. This could be confusing for the user if the user expects the print preview and the map to behave as one.
> 2. Keep the center point and zoom level from print preview. As the form factor differs between paper and screen the exact extent can not be displayed.  This could be confusing if user has recomposed the extent in print preview and the user expects that what happens in the print preview stays in the print preview.
> 
> Basically it depends on what the user expects and I think that the displayed behavior is the result of lack of decision what the user should expect.

It probably been like this since #1032 so not for always but a long time though.
